io-classes-mtl/io-classes-mtl.cabal

```diff
@@ -46,7 +46,7 @@ library
                       array,
                       mtl,
 
-                      io-classes ^>= 1.0.0.0,
+                      io-classes  >= 1.0 && < 1.2,
```

Comment:
There is no changelog entry for this change

Reply:
Yes, because I will just add a revision on hackage, no need to bump its version.